package.json

```diff
@@ -5607,6 +5607,11 @@
 				"title": "Create Patch...",
 				"category": "GitLens"
 			},
+			{
+				"command": "gitlens.graph.createPatch",
+				"title": "Create Patch...",
+				"category": "GitLens"
+			},
 			{
 				"command": "gitlens.createCloudPatch",
 				"title": "Create Patch...",
```

Comment:
We should create a `gitlens.graph.createCloudPatch` version like you did with `createPatch`

Reply:
done